src/logger.ts

```diff
@@ -29,8 +27,14 @@ export type LoggerOptions = {
 };
 
 export class Logger {
+  #secretsToMask: string[] = [];
```

Comment:
private fields https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#ecmascript-private-fields

Reply:
Oh! I didn't know that. TIL.

Wouldn't it be more readable to use the `private` keyword? It can even be `private readonly` as it is being declared on initialization and the value is not replaced, only modified with the push keyword.

Reply:
Never mind, [I just found out](https://stackoverflow.com/a/59641565/4330018). The hashtag keeps it private on runtime, while the `private` keyword keeps it private in compile time.